ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -9,7 +9,40 @@
 !
 event-handler {{ handler.name }}
 {%         if handler.trigger is arista.avd.defined %}
+{%             if handler.trigger is arista.avd.defined("on-maintenance")
+                  and handler.trigger_on_maintenance is arista.avd.defined
```

Comment:
no need for this since it is implicit in the check for .operation

Reply:
fixed